### ISSUE TITLE
It doesn't seem like the "You're feel like you're gonna have a bad time." text ever shows up.

### ISSUE THREAD

**Issue by FawfulBeanish** [Jcw87/c2-sans-fight#18]
It's a really small thing, but whenever you are in phase 1 and outside the special phase 2 text it always says the other "You felt your sins crawling on your back." and doesn't say the other text at any time. I wanted this to be fixed because I'm modding the text in game for my own demented purposes and I would like there to be two types of showable text on phase 1. I'm not sure how  or when the original game does it though.

**Comment by FawfulBeanish** [comment]
EDIT: It turns out that there is aquite a variety of text variations for phase 1, but I found this one in the game's code so I was wondering why it wouldn't show up.

**Comment by Jcw87** [comment]
I added all of the phase 1 messages at one point, but only 2 of them are in my code right now. The first one (You feel like you're going to have a bad time.) is immediately being overwritten by the second one (You felt your sins crawling on your back.), so you never see it. That's a bug, and I'm gonna fix it.

According to a gamemaker decompiler that was used on the original Undertale release, the following messages are displayed based on the player's KARMA at the end of an attack:

* You felt your sins crawling on your back. (0 - 9)
* You felt your sins weighing on your neck. (10 - 19)
* KARMA coursing through your veins. (20 - 29)
* Doomed to death of KARMA! (30 - 40) (40 is max)

IIRC, when I was testing it, I could never get the original game to display the last two, but triggering the 3rd one in my version was really easy. That's a pretty big discrepancy, so I figure that the logic deciding this must have changed in the update. The decompiler doesn't work with the new version, so I can't check to see if the conditions changed, or what they changed to. Without knowing what the new conditions may be, I decided to just leave out the other messages.

There's also a few other messages in the decompiled code, but their conditions are a little more cryptic:

* Just keep attacking. (hit attempts 0 - 3 and something else)
* Sans's movements grow a little wearier. (hit attempts 4 - 7 and something else)
* Sans's movements seem to be slower. (hit attempts 8 - 12 and something else)
* Felt like a turning point. (hit attempts 13+ and something else)

All of these have the same extra condition, but I don't know what the variable used in that condition means. With more time, I probably could have figured it out, but I was getting tired of working on these messages.
